opensearch-operator/mocks/github.com/Opster/opensearch-k8s-operator/opensearch-operator/pkg/reconcilers/k8s/mock_K8sClient.go

```diff
@@ -42,6 +42,10 @@ func (_m *MockK8sClient) EXPECT() *MockK8sClient_Expecter {
 func (_m *MockK8sClient) Context() context.Context {
 	ret := _m.Called()
 
+	if len(ret) == 0 {
+		panic("no return value specified for Context")
+	}
+
```

Comment:
Hey @salyh I assume all the added if conditions are generated code right ?

Reply:
yes, this because I updated mockery